src/IB/IBFEInstrumentPanel.cpp

```diff
@@ -0,0 +1,1067 @@
+
```

Comment:
Please use the standard file format from the rest of the library for implementation files. See, e.g., [IBInstrumentPanel.cpp](https://github.com/IBAMR/IBAMR/blob/master/src/IB/IBInstrumentPanel.cpp) for an example.